templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -191,5 +191,39 @@
 				</div>
 			{{end}}
 		{{end}}
+
+		<div class="ui divider"></div>
+		<span class="text"><strong>{{.i18n.Tr "repo.issues.due_date"}}</strong></span>
+		{{if gt .Issue.DeadlineUnix 0}}
+			<p>
+				<span class="octicon octicon-calendar"></span>
+				{{.Issue.DeadlineUnix.FormatShort}}
+				{{if .Issue.IsOverdue}}
+					<span style="color: red;">{{.i18n.Tr "repo.issues.due_date_overdue"}}</span>
+				{{end}}
+				<br/>
+				<a style="cursor:pointer;" onclick="toggleDuedateForm();"><i class="edit icon"></i>Edit</a> -
+				<a style="cursor:pointer;" onclick="deleteDueDate('{{$.RepoLink}}/issues/{{.Issue.Index}}/deadline/remove');"><i class="remove icon"></i>Remove</a>
```

Comment:
This can/should be standard link (`<a href="{{$.RepoLink}}/issues/{{.Issue.Index}}/deadline/remove">`), right?

Reply:
No, the js function sends a post request.

Reply:
I'll send a seperate pr after this one is merged which will add a seperate api endpoint for deadlines which then will be used for the update the deadline from the UI.

Reply:
Good to know you will improve this solution and add api endpoint. Until then, you should not simulate api endpoint by using js and standard page, ignoring anything it can return. For now, there is not reason to use js, you can add it later, in another PR when api would be ready for it.

Reply:
I've modifed the endpoint, deleting a deadline is now done via GET.